pyproject.toml

```diff
@@ -133,6 +134,7 @@ lint.ignore = [
   "E221",   # Multiple spaces before operator
   "E226",   # Missing whitespace around arithmetic operator
   "E241",   # Multiple spaces after ','
+  "PIE790", # flake8-pie: unnecessary-placeholder
```

Comment:
Just for the record, you're ignoring this because you dislike the rule? I mean, we could adjust to it. https://github.com/hugovk/Pillow/pull/130